examples/c/ex_backup.c

```diff
@@ -153,80 +254,188 @@ take_full_backup(WT_SESSION *session, int i)
         hdir = h;
     } else
         hdir = home_incr;
-    error_check(session->open_cursor(session, "backup:", NULL, NULL, &cursor));
+    if (i == 0) {
+        (void)snprintf(
+          buf, sizeof(buf), "incremental=(granularity=1M,enabled=true,this_id=\"ID%d\")", i);
+        error_check(session->open_cursor(session, "backup:", NULL, buf, &cursor));
+    } else
+        error_check(session->open_cursor(session, "backup:", NULL, NULL, &cursor));
 
+    count = 0;
+    alloc = FLIST_INIT;
+    flist = calloc(alloc, sizeof(FILELIST));
+    testutil_assert(flist != NULL);
     while ((ret = cursor->next(cursor)) == 0) {
         error_check(cursor->get_key(cursor, &filename));
+        error_check(process_file(&flist, &count, &alloc, filename));
+
+        /*
+         * If it is a log file, prepend the path for cp.
```

Comment:
since it's a comment, this should be copy not cp.

Reply:
I don't agree with this one - the `cp` refers to the specific `testutil_system` call below where we call `cp`.